templates/web/fixmystreet/footer.html

```diff
@@ -37,5 +37,10 @@
 
 <!-- [% INCLUDE 'debug_footer.html' %] -->
     </div> <!-- .wrapper -->
+
+    [% IF c.cobrand.moniker == 'fixmystreet' %]
+        [% INCLUDE 'footer-research-js.html' %]
+    [% END %]
+
```

Comment:
This one is outstanding being moved to a separate template (`footer_extra.html` to match the header one?), as mentioned at https://github.com/mysociety/fixmystreet/commit/9ca588ae0#commitcomment-9217074 .